diff -Nru convirt-0.9.6/src/convirt/core/platforms/xen/XenServerProxy.py convirt-0.9.6-patched/src/convirt/core/platforms/xen/XenServerProxy.py
--- convirt-0.9.6/src/convirt/core/platforms/xen/XenServerProxy.py	2008-11-17 08:21:10.000000000 -0200
+++ convirt-0.9.6-patched/src/convirt/core/platforms/xen/XenServerProxy.py	2009-01-27 02:42:28.000000000 -0200
@@ -25,6 +25,8 @@
 import xmlrpclib, socket, os, stat
 import SocketServer
 
+import sys
+sys.path.append("/usr/lib/xen-3.2-1/lib/python/") 
 
 import xen.xend.XendClient
 from xen.xend.XendLogging import log
